packages/devtools_app/lib/src/shared/memory/retainers.dart

```diff
@@ -34,6 +43,10 @@ ShortestRetainersResult findShortestRetainers({
   bool calculateSizes = true,
 }) {
   assert(refs(_sentinelIndex).isEmpty);
+  assert(
+    shallowSize(_sentinelIndex) <= 0,
```

Comment:
how about != 0 so we also catch cases where the size is greater than 0?

Reply:
Nope, it is correct, explained in comment: `Sentinel should have size 0 or -1 (not defined), but size is ${shallowSize(_sentinelIndex)}.`